### ISSUE TITLE
Unable to write pod event

### ISSUE THREAD

**Issue by supermar1010** [containers/libpod#5789]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

Starting pods fails with `ERRO[0000] unable to write pod event: "open : no such file or directory" `

**Steps to reproduce the issue:**

1. Not sure, for me it occurs when starting my containers

**Describe the results you received:**

Errors are thrown.

**Describe the results you expected:**

No errors are thrown.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:            1.8.2
RemoteAPI Version:  1
Go Version:         go1.13.6
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.13.6
  podman version: 1.8.2
host:
  BuildahVersion: 1.14.3
  CgroupVersion: v2
  Conmon:
    package: conmon-2.0.14-1.fc31.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.14, commit: 083a0be12178013d44ff51ceda3090ea741b6516'
  Distribution:
    distribution: fedora
    version: "31"
  IDMappings:
    gidmap:
    - container_id: 0
      host_id: 1002
      size: 1
    - container_id: 1
      host_id: 231072
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1002
      size: 1
    - container_id: 1
      host_id: 231072
      size: 65536
  MemFree: 6634127360
  MemTotal: 8342446080
  OCIRuntime:
    name: crun
    package: crun-0.13-1.fc31.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.13
      commit: e79e4de4ac16da0ce48777afb72c6241de870525
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  SwapFree: 0
  SwapTotal: 0
  arch: amd64
  cpus: 2
  eventlogger: file
  hostname: v220190910461598152
  kernel: 5.5.15-200.fc31.x86_64
  os: linux
  rootless: true
  slirp4netns:
    Executable: /usr/bin/slirp4netns
    Package: slirp4netns-0.4.0-20.1.dev.gitbbd6f25.fc31.x86_64
    Version: |-
      slirp4netns version 0.4.0-beta.3+dev
      commit: bbd6f25c70d5db2a1cd3bfb0416a8db99a75ed7e
  uptime: 1h 18m 26.79s (Approximately 0.04 days)
registries:
  search:
  - docker.io
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - registry.centos.org
  - quay.io
store:
  ConfigFile: /home/jc2/.config/containers/storage.conf
  ContainerStore:
    number: 9
  GraphDriverName: overlay
  GraphOptions:
    overlay.mount_program:
      Executable: /usr/bin/fuse-overlayfs
      Package: fuse-overlayfs-0.7.8-1.fc31.x86_64
      Version: |-
        fusermount3 version: 3.6.2
        fuse-overlayfs: version 0.7.8
        FUSE library version 3.6.2
        using FUSE kernel interface version 7.29
  GraphRoot: /home/jc2/.local/share/containers/storage
  GraphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  ImageStore:
    number: 5
  RunRoot: /run/user/1002
  VolumePath: /home/jc2/.local/share/containers/storage/volumes

```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
Name         : podman
Epoch        : 2
Version      : 1.8.2
Release      : 2.fc31
Architecture : x86_64
Size         : 59 M
Source       : podman-1.8.2-2.fc31.src.rpm
Repository   : @System
From repo    : updates
Summary      : Manage Pods, Containers and Container Images
URL          : https://podman.io/
License      : ASL 2.0
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Output from podman-compose
```
podman pod create --name=backend --share net -p 18081:8081 -p 127.0.0.1:10000:3000
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
1a17740232c30444f4922284e2b0c27c1b87c2c4ca6f4c6895e14b44ed9debfc
0
podman run --name=backend_node_1 -d --pod=backend --label io.podman.compose.config-hash=123 --label io.podman.compose.project=backend --label io.podman.compose.version=0.0.1 --label com.docker.compose.container-number=1 --label com.docker.compose.service=node --mount type=bind,source=/home/jc2/podman/jc2/backend/./jodelcity2backend/,destination=/home/node/app,bind-propagation=Z --add-host node:127.0.0.1 --add-host backend_node_1:127.0.0.1 --add-host db:127.0.0.1 --add-host mongoJC2:127.0.0.1 --add-host mongo-express:127.0.0.1 --add-host backend_mongo-express_1:127.0.0.1 -w /home/node/app node:latest bash -c cp /usr/share/zoneinfo/Europe/Berlin /etc/localtime && npm install && npm install pm2 -g && pm2-runtime start index.js
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0001] unable to write pod event: "open : no such file or directory" 
ERRO[0001] unable to write pod event: "open : no such file or directory" 
213160b3bc9f7d271d02e728bba3bd811d2814bc19ea3d2cc7c59241dadcee10
0
podman run --name=mongoJC2 -d --pod=backend --label io.podman.compose.config-hash=123 --label io.podman.compose.project=backend --label io.podman.compose.version=0.0.1 --label com.docker.compose.container-number=1 --label com.docker.compose.service=db --mount type=bind,source=/home/jc2/podman/jc2/backend/./datadir,destination=/data/db --add-host node:127.0.0.1 --add-host backend_node_1:127.0.0.1 --add-host db:127.0.0.1 --add-host mongoJC2:127.0.0.1 --add-host mongo-express:127.0.0.1 --add-host backend_mongo-express_1:127.0.0.1 mongo
ERRO[0000] unable to write volume event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
f49a5a4ecb8e3fb7969360e83205204bc6a4e5e32d34c24396868528ddd024a6
0
podman run --name=backend_mongo-express_1 -d --pod=backend --label io.podman.compose.config-hash=123 --label io.podman.compose.project=backend --label io.podman.compose.version=0.0.1 --label com.docker.compose.container-number=1 --label com.docker.compose.service=mongo-express -e ME_CONFIG_BASICAUTH_USERNAME=admin -e ME_CONFIG_BASICAUTH_PASSWORD=LKJhafdssk!hDhj -e ME_CONFIG_MONGODB_SERVER=db --add-host node:127.0.0.1 --add-host backend_node_1:127.0.0.1 --add-host db:127.0.0.1 --add-host mongoJC2:127.0.0.1 --add-host mongo-express:127.0.0.1 --add-host backend_mongo-express_1:127.0.0.1 mongo-express
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
0132b0e131a7ebbb2cbdbc2069b1735c9cbbc8cbf83ed7a9e3e59626c5262250
0
```

**Comment by mheon** [comment]
Can you provide `~/.config/containers/libpod.conf`

**Comment by supermar1010** [comment]
What also might be worth noting is, that I always have to set `XDG_RUNTIME_DIR=/run/user/1002` before that it tries to use the id 1000, which fails with a permission denied, but this might also be a problem with the account I'm using, not sure if this is related, just wanted to mention it :)

```volume_path = "/home/jc2/.local/share/containers/storage/volumes"
image_default_transport = "docker://"
runtime = "runc"
runtime_supports_json = ["crun", "runc"]
runtime_supports_nocgroups = ["crun"]
conmon_path = ["/usr/libexec/podman/conmon", "/usr/local/libexec/podman/conmon", "/usr/local/lib/podman/conmon", "/usr/bin/conmon", "/usr/sbin/conmon", "/usr/local/bin/conmon", "/usr/local/sbin/conmon", "/run/current-system/sw/bin/conmon"]
conmon_env_vars = ["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
cgroup_manager = "cgroupfs"
init_path = ""
static_dir = "/home/jc2/.local/share/containers/storage/libpod"
tmp_dir = "/run/user/1002/libpod/tmp"
max_log_size = -1
no_pivot_root = false
cni_config_dir = "/etc/cni/net.d/"
cni_plugin_dir = ["/usr/libexec/cni", "/usr/lib/cni", "/usr/local/lib/cni", "/opt/cni/bin"]
infra_image = "k8s.gcr.io/pause:3.1"
infra_command = "/pause"
enable_port_reservation = true
label = true
network_cmd_path = ""
num_locks = 2048
lock_type = "shm"
events_logger = "journald"
events_logfile_path = ""
detach_keys = "ctrl-p,ctrl-q"
SDNotify = false

[runtimes]
  crun = ["/usr/bin/crun", "/usr/local/bin/crun"]
  runc = ["/usr/bin/runc", "/usr/sbin/runc", "/usr/local/bin/runc", "/usr/local/sbin/runc", "/sbin/runc", "/bin/runc", "/usr/lib/cri-o-runc/sbin/runc", "/run/current-system/sw/bin/runc"]
```

**Comment by supermar1010** [comment]
Well this issue solved itself after recreating that user account, I'm not sure why

**Comment by rhatdan** [comment]
I would figure this was permissions problems on /run or something like that.

**Comment by tigerpaws2** [comment]
I am seeing exactly the same thing, but with podman 1.9.0.


```
[mike@polly ~]$ podman run jupyterlab -p 8888:8889
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
ERRO[0000] unable to write pod event: "open : no such file or directory" 
```
Running podman with debug, it turns out that this is the real error: 

```
DEBU[0000] Found deprecated file /home/mike/.config/containers/libpod.conf, please remove. Use /home/mike/.config/.config/containers/containers.conf to override defaults. 
DEBU[0000] Reading configuration file "/home/mike/.config/containers/libpod.conf" 

```
I deleted libpod.conf, and voila, no more pod event errors.

**Comment by anthonyjmartinez** [comment]
@tigerpaws2 - thanks for doing the debug for us all. Always nice when the first search result contains the answer.

For me this started after upgrading from F31 to F32.

**Comment by mheon** [comment]
I think this is a separate bug (containers.conf related). Hopefully we'll get it fixed in 1.9.2 or 1.9.3.
